website/docs/getting-started/contribution/partials/icons.md

```diff
@@ -30,7 +28,7 @@ Contributing is not required to get new icons added to the library, but HashiCor
 - ![](/assets/foundations/flight-icons/icon-dimensions-helper-3.png) 2px
 - ![](/assets/foundations/flight-icons/icon-dimensions-helper-4.png) 1.5px
 
-#### 16px
+### 16px
 
 ![](/assets/foundations/flight-icons/icon-dimensions-sq-16.png =80x80)
```

Comment:
@didoo I see these images in folders named after their path in markdown. does it make sense for me to create a `/assets/getting-started/contribution/icons` folder to house these images and move them over?

(I know i have a homework assignment for this but I haven't gotten to it yet)